skore/src/skore/sklearn/_plot/metrics/precision_recall_curve.py

```diff
@@ -910,3 +912,68 @@ def _compute_data_for_display(
             ml_task=ml_task,
             report_type=report_type,
         )
+
+    def frame(self, with_average_precision: bool = False) -> DataFrame:
+        """Get the data used to create the precision-recall curve plot.
+
+        Parameters
+        ----------
+        with_average_precision : bool, default=False
+            Whether to include the average precision column in the returned DataFrame.
+
+        Returns
+        -------
+        DataFrame
+            A DataFrame containing the precision-recall curve data with columns
+            depending on the report type:
+
+            - estimator_name: Name of the estimator (when comparing estimators)
+            - split_index: Cross-validation fold ID (when doing cross-validation)
+            - label: Class label (for multiclass-classification)
+            - threshold: Decision threshold
+            - precision: Precision score at threshold
+            - recall: Recall score at threshold
+            - average_precision: average precision (when `with_average_precision=True`)
+
+        Examples
+        --------
+        >>> from sklearn.datasets import load_breast_cancer
+        >>> from sklearn.linear_model import LogisticRegression
+        >>> from skore import train_test_split, EstimatorReport
+        >>> X, y = load_breast_cancer(return_X_y=True)
+        >>> split_data = train_test_split(X=X, y=y, random_state=0, as_dict=True)
+        >>> classifier = LogisticRegression(max_iter=10_000)
+        >>> report = EstimatorReport(classifier, **split_data)
+        >>> display = report.metrics.precision_recall()
+        >>> df = display.frame()
+        """
+        df = (
+            self.precision_recall
+            if not with_average_precision
+            else self.precision_recall.merge(self.average_precision)
+        )
+        # If done, the merge between the precision-recall curve and the average
+        # precision is done without specifying the columns to merge on, hence done on
+        # all columns that are present in both DataFrames.
+        # In this case, the common columns are all columns but not the one containing
+        # the statistics.
```

Comment:
You previous code was better because we can add the comment in the associated branch. Otherwise, it is hard to see at a first glance to what it applies to.

```suggestion
        if with_average_precision:
            # If done, the merge between the precision-recall curve and the average
            # precision is done without specifying the columns to merge on, hence done on
            # all columns that are present in both DataFrames.
            # In this case, the common columns are all columns but not the one containing
            # the statistics.
            df = self.precision_recall.merge(self.average_precision)
        else:
            df = self.precision_recall
```

Reply:
OK, try to push the new version to check if it raises an error in which case we can try to bypass it. Be aware that you can always bypass pre-commit when committing with `git commit -am"toto" --no-verify`